.rubocop.yml

```diff
@@ -30,3 +30,6 @@ Metrics/ClassLength:
 Performance/FixedSize:
   Exclude:
     - 'test/**/*'
+
+Style/AndOr:
+  EnforcedStyle: conditionals
```

Comment:
I don't care too much, I like the shorthand 'redirect and return' but I'm ok with moving that to 2 lines